backend/src/test/java/com/woowacourse/gongcheck/core/application/JobServiceTest.java

```diff
@@ -65,272 +70,518 @@ class JobServiceTest {
     @Autowired
     private RunningTaskRepository runningTaskRepository;
 
-    @Test
-    void Job_목록을_조회한다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
-        Space space = spaceRepository.save(Space_생성(host, "잠실"));
-        Job job1 = Job_생성(space, "오픈");
-        Job job2 = Job_생성(space, "청소");
-        Job job3 = Job_생성(space, "마감");
-        jobRepository.saveAll(List.of(job1, job2, job3));
-
-        JobsResponse result = jobService.findJobs(host.getId(), space.getId());
-
-        assertThat(result.getJobs()).hasSize(3);
-    }
-
-    @Test
-    void 존재하지_않는_Host로_Job_목록을_조회할_경우_예외를_던진다() {
-        assertThatThrownBy(() -> jobService.findJobs(0L, 1L))
-                .isInstanceOf(NotFoundException.class)
-                .hasMessage("존재하지 않는 호스트입니다.");
-    }
+    @Nested
+    class findJobs_메소드는 {
+
+        @Nested
+        class Job_목록이_세개가_있는_경우 {
+
+            private Host host;
+            private Space space;
+
+            @BeforeEach
+            void setUp() {
+                host = hostRepository.save(Host_생성("1234", 1234L));
+                space = spaceRepository.save(Space_생성(host, "잠실"));
+                Job job_1 = Job_생성(space, "오픈");
+                Job job_2 = Job_생성(space, "청소");
+                Job job_3 = Job_생성(space, "마감");
+                jobRepository.saveAll(List.of(job_1, job_2, job_3));
```

Comment:
```suggestion
                jobRepository.saveAll(List.of(Job_생성(space, "오픈"), Job_생성(space, "청소"), Job_생성(space, "마감")));
```
아까 이야기한대로 현재는 지금과 같이 수정하는것이 좋을 것 같네요
만약 구체적인 검증이 필요하다면 context class(Job_목록이_세개가_있는_경우) 변수로 값을 두고 세팅하는걸로!